projects/managed-hubs/support.md

```diff
@@ -200,39 +219,39 @@ When a new ticket lands in Freshdesk under the support group and it is not an in
-   
+
    :::{admonition} What does `critical` mean?
 
-   We recognize there might be some support-related issues that do not count as [incidents](incidents:what), but 
-   they need a quick resolution (inside the current sprint window) because they are impacting the execution of 
+   We recognize there might be some support-related issues that do not count as [incidents](incidents:what), but
+   they need a quick resolution (inside the current sprint window) because they are impacting the execution of
    desired or existing workflows (degraded experience) for our communities.
    Examples of those sorts of issues (requests) are:
      * Image refs updates
      * Profile updates
      * User storage limitations
      * Grafana (and Prometheus) failures
-   
+
    Additionally and depending on the nature AND context of the issue (request):
      * Access to specific buckets
      * Authentication and authorization updates
    :::
-   
+
    The support steward **should** self-assign the `critical` issue and work on it immediately (this is now outside of the 30-minute timebox described in step 2).
-   
+
    If the support stewards (both of them) do not have the capacity to resolve the `critical` issue (ie. working on another `critical` issue, being out of their working time, etc.), they should ping the **Engineering Manager** (or the delegated person) so they can secure resources to resolve that issue on the fly (see step 7 below).
-   
+
    The support steward **should not** work on issues with impact lower than `critical` (unless they are assigned as part of the "planned" reactive work in the context of a running sprint (see step 6 below).
 
-4. `Who: Partnerships representative and the Engineering Manager (or respective delegates)`
+5. `Who: Partnerships representative and the Engineering Manager (or respective delegates)`
 
    **Revisit the impact metadata**. Once a week (at minimum) the [support view in the **Eng & Prod** board](https://github.com/orgs/2i2c-org/projects/22/views/47) should be revisited to validate the impact level on support-related issues. Currently, we allocate a 30-minute working session every Wednesday (open to everyone to participate) to perform such impact revision and further prioritization ("planned" reactive) every other week (see step 7 for more details).
-     
-5. `Who: Support steward`
 
-   **Add a reference/link to the created engineering issue inside the Freshdesk ticket**. You can use an internal note or make it public when you communicate back to the Community Representative in step 6. Also, move the status of the ticket to the "Pending" state.
-   
 6. `Who: Support steward`
 
+   **Add a reference/link to the created engineering issue inside the Freshdesk ticket**. You can use an internal note or make it public when you communicate back to the Community Representative in step 6. Also, move the status of the ticket to the "Pending" state.
```

Comment:
```suggestion
   **Add a reference/link to the created engineering issue inside the Freshdesk ticket and set ticket status to Pending**. You can use a private note or make it public when you communicate back to the Community Representative in the next step.
```

Reply:
although this made me realize that I'm going to have to touch this anyway, as the step names have changed. So I'll just incorporate your suggestion into it too @consideRatio when i edit it!